src/type-layout.md

```diff
@@ -332,13 +332,14 @@ Like all ways to create undefined behavior in safe Rust, this is a bug.
 
 ### The `transparent` Representation
 
-The `transparent` representation can only be used on `struct`s that have:
+The `transparent` representation can only be used on a [`struct`][structs] that has,
+or an [`enum`][enumerations] with a single variant which has:
```

Comment:
I was initially a little confused with how this is worded (it seemed a little awkward).  Instead of splitting it, maybe word it something like this?

```
The `transparent` representation can only be used on a [`struct`][structs] or
single variant [`enum`][enumerations] that has:
```

I'm guessing the intent was to avoid ambiguity of whether or not the two rules apply to both structs and enums?  I'm not terribly strong at grammar, but I would say without a comma it is not an independent clause, and should be considered together.

Reply:
The idea was to avoid the implication that the attribute is placed on the variant as opposed to the `enum`. I'll update to your wording.